tests/__test__/logout.test.ts

```diff
@@ -0,0 +1,23 @@
+import {logout } from "../../api/logout"; // Assuming the file containing the logout function is named logout.js
+import {account } from "../../api/logout";
+import jest from "jest";
+
+jest.mock('account', () => ({
+  account: {
+    deleteSession: jest.fn(),
+  }
+}));
+
+  it('calls deleteSession with "current" parameter and returns session', async () => {
```

Comment:
For this test, we can mock when it passes and fails. But we need an integration to test that it actually calls appwrite as well. This won't work unless we have the proper credentails.